src/solver_cvodes.cpp

```diff
@@ -118,6 +118,8 @@ void CVodeSolver::initSteadystate(const realtype /*t0*/, const AmiVector &/*x0*/
        as re-calling init would unset solver settings. */
     auto cv_mem = static_cast<CVodeMem>(solver_memory_.get());
     cv_mem->cv_f = fxBdot_ss;
+
```

Comment:
maybe add a comment that both functions must be set for simulation to work properly and that just setting `cv_mem->cv_f` will result in the previous function being used as alternate Nls function?